system-tests/src/test/java/org/eclipse/edc/samples/transfer/ConsumerPullSampleTestCommon.java

```diff
@@ -0,0 +1,222 @@
+                "    \"contractPolicyId\": \"aPolicy\",\n" +
+                "    \"assetsSelector\": []\n" +
+                "}";
+
+        ContractOfferId=given()
+                .header("Content-Type", "application/json")
+                .contentType(ContentType.JSON)
+                .body(requestBody)
+                .when()
+                .post(MANAGEMENT_API_URL + "/management/v2/contractdefinitions")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("@id", not(emptyString()))
+                .extract()
+                .jsonPath()
+                .get("@id");
+    }
+
+    public void fetchCatalog(){
+        String requestBody = "{\n" +
+                "    \"@context\": {\n" +
+                "        \"edc\": \"https://w3id.org/edc/v0.0.1/ns/\"\n" +
+                "    },\n" +
+                "    \"providerUrl\": \"http://localhost:19194/protocol\",\n" +
+                "    \"protocol\": \"dataspace-protocol-http\"\n" +
+                "}";
+
+        given()
+                .header("Content-Type", "application/json")
+                .contentType(ContentType.JSON)
+                .body(requestBody)
+                .when()
+                .post("http://localhost:29193/management/v2/catalog/request")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body("@id",not(emptyString()))
+                .body("@type",equalTo("dcat:Catalog"));
+    }
+
+    public void initiateContractNegotiation(){
+        String requestBody = "{\n" +
+                "    \"@context\": {\n" +
+                "        \"edc\": \"https://w3id.org/edc/v0.0.1/ns/\",\n" +
+                "        \"odrl\": \"http://www.w3.org/ns/odrl/2/\"\n" +
+                "    },\n" +
+                "    \"@type\": \"NegotiationInitiateRequestDto\",\n" +
+                "    \"connectorId\": \"provider\",\n" +
+                "    \"connectorAddress\": \"http://localhost:19194/protocol\",\n" +
+                "    \"consumerId\": \"consumer\",\n" +
+                "    \"providerId\": \"provider\",\n" +
+                "    \"protocol\": \"dataspace-protocol-http\",\n" +
+                "    \"offer\": {\n" +
+                "        \"offerId\": \"MQ==:YXNzZXRJZA==:YTc4OGEwYjMtODRlZi00NWYwLTgwOWQtMGZjZTMwMGM3Y2Ey\",\n" +
+                "        \"assetId\": \"assetId\",\n" +
+                "        \"policy\": {\n" +
+                "            \"@id\": \"MQ==:YXNzZXRJZA==:YTc4OGEwYjMtODRlZi00NWYwLTgwOWQtMGZjZTMwMGM3Y2Ey\",\n" +
+                "            \"@type\": \"Set\",\n" +
+                "            \"odrl:permission\": [],\n" +
+                "            \"odrl:prohibition\": [],\n" +
+                "            \"odrl:obligation\": [],\n" +
+                "            \"odrl:target\": \"assetId\"\n" +
+                "        }\n" +
+                "    }\n" +
+                "}";
+
+        ContractNegotiationId=given()
```

Comment:
```suggestion
        return = given()
```
You may want to directly return the property you need in the test file. (same goes for other methods)